packages/ember-metal/lib/watch_key.js

```diff
@@ -102,13 +104,19 @@ export function unwatchKey(obj, keyName, meta) {
               enumerable: true,
               value: val
             });
-            m.deleteFromValues(keyName);
+
+            var m = peekMeta(obj);
```

Comment:
This definitely seems correct, but shadowing the outer 'm' *feels* odd to me. Can we use either the variable declared above (just inside the main unwatchKey function) or pick a different variable name than 'm'?